docs/xtrabackup-versioning-scheme.md

```diff
@@ -0,0 +1,9 @@
+### Percona XtraDB Cluster 5.7 versioning scheme:
+
+Here is what the numbers in the product version specification mean:
+
+For example, in Percona XtraDB Cluster 5.7.39-31.61
+
+* `5.7.39-31` - the version of Percona Server for MySQL that Percona XtraDB Cluster is based on
+
+* `61` - a sequential number that increases every time Percona releases a new version of Percona XtraDB Cluster
```

Comment:
This is the next odd number (odd for historical reasons, when we also had 5.6 series which used even numbers). 
Please check https://confluence.percona.com/pages/viewpage.action?pageId=104088973

Reply:
Updated